topicgenerator/src/main/java/io/dockstore/topicgenerator/client/cli/TopicGeneratorClient.java

```diff
@@ -84,48 +88,37 @@ public static void main(String[] args) {
 
+            if ("generate-topics".equals(jCommander.getParsedCommand())) {
                 // Read CSV file
-                Iterable<CSVRecord> entriesCsvRecords = null;
-                try {
-                    final Reader entriesCsv = new FileReader(generateTopicsCommand.getEntriesCsvFilePath());
-                    CSVFormat csvFormat = CSVFormat.DEFAULT.builder()
-                            .setHeader(InputCsvHeaders.class)
-                            .setSkipHeaderRecord(true)
-                            .setTrim(true)
-                            .build();
-                    entriesCsvRecords = csvFormat.parse(entriesCsv);
-                } catch (IOException e) {
-                    exceptionMessage(e, "Unable to read input CSV file", IO_ERROR);
-                }
-
-                final TopicGeneratorClient topicGeneratorClient = new TopicGeneratorClient();
-                topicGeneratorClient.generateTopics(topicGeneratorConfig, entriesCsvRecords);
+                topicGeneratorClient.generateTopics(topicGeneratorConfig, generateTopicsCommand.getEntriesCsvFilePath());
+            } else if ("upload-topics".equals(jCommander.getParsedCommand())) {
+                // Read CSV file
+                topicGeneratorClient.uploadTopics(topicGeneratorConfig, uploadTopicsCommand.getAiTopicsCsvFilePath());
             }
         }
     }
 
     /**
      * Generates a topic for public entries by asking the GPT-3.5-turbo-16k AI model to summarize the content of the entry's primary descriptor.
      * @param topicGeneratorConfig
-     * @param entriesCsvRecords
+     * @param inputCsvFilePath
      */
-    private void generateTopics(TopicGeneratorConfig topicGeneratorConfig, Iterable<CSVRecord> entriesCsvRecords) {
+    private void generateTopics(TopicGeneratorConfig topicGeneratorConfig, String inputCsvFilePath) {
         final ApiClient apiClient = setupApiClient(topicGeneratorConfig.dockstoreServerUrl());
         final Ga4Ghv20Api ga4Ghv20Api = new Ga4Ghv20Api(apiClient);
         final OpenAiService openAiService = new OpenAiService(topicGeneratorConfig.openaiApiKey());
         final String outputFileName = OUTPUT_FILE_PREFIX + "_" + AI_MODEL + "_" + Instant.now().truncatedTo(ChronoUnit.SECONDS).toString().replace("-", "").replace(":", "") + ".csv";
+        final Iterable<CSVRecord> entriesCsvRecords = readCsvFile(inputCsvFilePath, GenerateTopicsCommand.InputCsvHeaders.class);
 
         try (CSVPrinter csvPrinter = new CSVPrinter(new FileWriter(outputFileName, StandardCharsets.UTF_8), CSVFormat.DEFAULT.builder().setHeader(OutputCsvHeaders.class).build())) {
             for (CSVRecord entry: entriesCsvRecords) {
-                final String trsId = entry.get(InputCsvHeaders.trsId);
-                final String versionId = entry.get(InputCsvHeaders.version);
+                final String trsId = entry.get(GenerateTopicsCommand.InputCsvHeaders.trsId);
```

Comment:
The lowercase enum value names confuse me every time.  No need to change, but I needed to vent that. :)  It would be great if the apache csv library calculated the column name as the `toString()` on the enum value, so the enum names themselves could be capitalized correctly (with a little extra work/code).